root/usr/bin/discovery-fetch-extensions

```diff
@@ -70,7 +70,7 @@ end
 
 # Write out environment files for subsequent paths and systemd units to use
 File.open('/etc/default/discovery','w') do |f|
-  f.write("# EnvironmentFile generated by discovery-setup.service\n")
+  f.write("# EnvironmentFile generated by discovery-fetch-extensions.service\n")
   f.write(env_append('PATH','/opt/extension/bin'))
   f.write(env_append('FACTERLIB','/usr/share/fdi/facts:/opt/extension/facts'))
   f.write(env_append('LD_LIBRARY_PATH','/opt/extension/lib'))
```

Comment:
Do we still need to create this file on-the-fly? I mean, can't we hardcode it into the image since we don't change it and there is no path hooked to it?